doc/changes/changes_6.0.0.md

```diff
@@ -0,0 +1,34 @@
+# Exasol Virtual Schema 6.0.0, released 2021-12-??
+
+Code name: Support native Exasol data types
```

Comment:
Maybe add a note that we should not release before the JDBC driver is released. If you add it to the first line, everyone will read it before updating the date.